src/Performance/Minibench.ss

```diff
@@ -0,0 +1,32 @@
+;; -*- mode: scheme -*-
+
+(library (Performance.Minibench foreign)
+  (export timeNs
+          gc
+          toFixed
+  )
+
+  (import (only (rnrs base) define let lambda quote)
+          (only (chezscheme)
+            collect format time-difference current-time time-nanosecond fixnum->flonum)
+  )
+
+  ;; timeNs :: (Unit -> a) -> Effect Number
+  ;; run a function accepting a unit and obtain the
+  ;; execution time in nanoseconds.
```

Comment:
I think there comments are not necessary. Any docblocks should be in the purs definitions.

Reply:
What convention is this referencing?

Reply:
Just that in general it doesn't make sense to add generic documentation about a purescript function into the foreign definition. For example it won't show up in the API docs in Pursuit, among other reasons.

Reply:
No, but `timeNs` isn't really that descriptive of a name - I don't really see the problem with comments as they improve maintainability. Just because people don't usually put documentation inside of a foreign definition does not mean its a bad thing - especially when the foreign definition doesn't provide type signatures/compile time guarantees that Purescript does.

Reply:
This function is already documented in the `.purs` file, it's easier to maintain the documentation in one place than possibly more than two if you count all the foreign implementations.

The second point is that comments like this might make the code more _difficult_ to maintain if they get outdated. This comment for example was already outdated, because of the signature change.

It is valuable though to add comments if for example here there would be something in particular that the scheme implementation does.